docs/source/release_notes/release-notes.rst

```diff
@@ -30,15 +30,21 @@ Changelog
   :hidden:
 
 =======
-develop
+v0.37.0
 =======
 
+29 Mar 2017
+
 .. list-table::
     :widths: 5 40
     :header-rows: 1
 
     *    - Type
          - Change
+    *    - |fixed|
+         - Fixed an issue where a ``MultipleRunningTimestampServicesError`` would be ignored, resulting in a state where two timestamp services would be able to simultaneously hand out timestamps.
```

Comment:
What does this mean for people who don't have this fix? Does this mean I could have silent data corruption? I'd try to communicate how likely this is to happen, or how serious of an issue this is.